llvm/utils/lit/tests/Inputs/shtest-env/env-args-none.txt

```diff
@@ -1 +1,30 @@
-# RUN: env
+# Check default environment.
```

Comment:
I think it would be useful to show what happens to variables that aren't set or unset on the command-line, in cases where other variables are. The easiest way to do this would be to add an additional environment variable to the default set and then just check it has its original value in every test case.